### DISCUSSION TITLE
Why no HTTP/3 Support for RHEL Based Systems?

### DISCUSSION THREAD

**Discussion by akama-aka** [openssl/openssl#24131] [Q&A] [answered]
Hi, I wanted to install cloudflare/quiche for my webserver today but I realised that OpenSSL does not support HTTP/3 for RHEL based systems. How come it is not available for RHEL based systems yet?

**Comment by nhorman** [comment]
Thats a question for the RHEL maintainers, as to why they didn't enable it.  @beldmit can you point @akama-aka in the right direction here please?

**Comment by beldmit** [comment]
Red Hat ships OpenSSL 3.0.x in RHEL 9 series that hardly support HTTP/3 (QUIC-based, I presume)

**Reply by akama-aka** [reply]
Okay, that means just NGINX needs then to implement it into their Versions right?

**Reply by beldmit** [reply, marked as answer]
I vaguely remember that there was some nginx work enabling http/3 
Probably this is a relevant link:

https://mailarchive.ietf.org/arch/msg/quic/M9pkSGzTSHunNC1yeySaB3irCVo/

**Comment by t8m** [comment]
This is not the right discussion forum to ask vendor-specific questions.
